src/Fmt_ast.ml

```diff
@@ -1081,6 +1081,16 @@ and fmt_body c ?ext ({ast= body} as xbody) =
         $ Cmts.fmt_after c pexp_loc )
   | _ -> (noop, fmt "@ " $ fmt_expression c ~eol:(fmt "@;<1000 0>") xbody)
 
+and fmt_assign_colon c =
```

Comment:
Minor comment: These functions do not need to be in the recursive declarations and could be before any functions working with the AST.